### ISSUE TITLE
id 135649373

### ISSUE THREAD

**Issue by nigelhorne** [whosonfirst-data/whosonfirst-data#1331]
https://spelunker.whosonfirst.org/id/135649373/ doesn't look right:

Firstly,  wof:name is set to the name of the borough it's in, rather than name, which I presume should be "Randall Close Day Centre".
Secondly, the addr:postcode field is prefaced by the string 'greater;'.

**Comment by nvkelso** [comment]
That's a `venue` record which generally don't receive as much TLC as the rest of the placetypes. My guess is something like Libpostal garbled it along the way, or the original SimpleGeo source was garbled.

**Comment by thisisaaronland** [comment]
Yes, there is a long-standing item on the TODO list to re-parse all of the addresses (in venues) using the latest version of libpostal. All things being equal that might happen in the month or two.
